util/mkdef.pl

```diff
@@ -386,7 +386,9 @@ sub writer_VMS {
 _____
 
     if (defined $version) {
-        my ($libvmajor, $libvminor) = $version =~ /^(\d+)_(\d+)$/;
+        $version =~ /^(\d+)\.(\d+)\.(\d+)/;
+        my $libvmajor = $1;
+        my $libvminor = $2 * 100 + $3;
```

Comment:
I didn't try to understand _why_ the minor and patch version need to be coded into `$libvminor` in this way. But the perl code looks good to me. So I'm just assuming that you know what you are doing... ;-)

Reply:
I do :-)